### PR TITLE
Connecting dots  graphs using cpp

### PR DESCRIPTION
**Pull request by anshika272**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

we have a board of size NxM. Each cell in the board is a coloured dot. There exist only 26 colours denoted by uppercase Latin characters (i.e. A,B,...,Z). we want to play a game. The key of this game is to find a cycle that contain dots of same colour. Formally, we call a sequence of dots d1, d2, ..., dk a cycle if and only if it meets the following condition:These k dots are different: if i ≠ j then di is different from dj.k is at least 4.All dots belong to the same colour.For all 1 ≤ i ≤ k - 1: di and di + 1 are adjacent. Also, dk and d1 should also be adjacent. Cells x and y are called adjacent if they share an edge.Our task is to determine if there exists a cycle on the board.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #7288

### CODE REVIEW

**Comment by anshika272** [comment]
@siddharth25pandey pls review my PR.

File: C-Plus-Plus/graphs/Connecting _Dots-Graphs_using_cpp.cpp [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+These k dots are different: if i ≠ j then di is different from dj.
+k is at least 4.
+All dots belong to the same colour.
+For all 1 ≤ i ≤ k - 1: di and di + 1 are adjacent. Also, dk and d1 should also be adjacent. Cells x and y are called adjacent if they share an edge.
+Our task is to determine if there exists a cycle on the board.
+*/
+
+
+#include <iostream>
+#include <vector>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/graphs/Connecting _Dots-Graphs_using_cpp.cpp]: Use single header file **bits/stdc**

File: C-Plus-Plus/graphs/Connecting _Dots-Graphs_using_cpp.cpp [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            if(ans){
+                return 1;
+            }
+        }
+    }
+    return 0;
+}
+
+int main() {
+    int n, m;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/graphs/Connecting _Dots-Graphs_using_cpp.cpp]: Use a good naming convention

File: C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp [hunk 3]
```diff
@@ -2,12 +2,12 @@
 i.e. find a node with a value just greater than n.
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp]: Remove the entire Redundant file

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@anshika272 Small changes are required, rest is fine

File: C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp (hunk 3 above)
Comment (anshika272) [reply C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp]: @siddharth25pandey i merged this file next larger element by mistake. its not related to this PR. this code has already been merged. is there any way we can delete it from this PR.

Comment (siddharth25pandey) [reply C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp]: @anshika272 Yup, you can do in two ways, first one through command line (terminal) & second way is removing the file manually from Github GUI. Another way is to create a Fresh new PR if earlier two ways are not easy for you then.

Comment (anshika272) [reply C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp]: okay ill try it

**Comment by anshika272** [comment]
@siddharth25pandey  please check the changes

**Review by siddharth25pandey — APPROVED** [review]
@anshika272 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @anshika272 for her code

**Pull request merged**